src/main/java/com/travel/pay/Main.java

```diff
@@ -0,0 +1,89 @@
+package com.travel.pay;
+
+import com.travel.pay.facade.TravelTourFacade;
+import io.vavr.Tuple;
+import io.vavr.control.Try;
+
+import java.util.Scanner;
+
+import static io.vavr.API.*;
+import static org.apache.commons.lang3.math.NumberUtils.toInt;
+
+class Main {
```

Comment:
Seems like an IDE suggestion? :) Avoid using package level visibility for common scenarios.